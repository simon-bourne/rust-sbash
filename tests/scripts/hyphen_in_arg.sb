pub fn test-fn(hyphenated-arg) {
}
