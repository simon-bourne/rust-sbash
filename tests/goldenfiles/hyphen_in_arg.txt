0: at line 0, in Tag:
pub fn test-fn(hyphenated-arg) {
                         ^

1: at line 0, in function signature:
pub fn test-fn(hyphenated-arg) {
       ^

2: at line 0, in function:
pub fn test-fn(hyphenated-arg) {
^

3: at line 0, in ManyTill:
pub fn test-fn(hyphenated-arg) {
^

4: at line 0, in script:
pub fn test-fn(hyphenated-arg) {
^

